docs/TroubleShooting.md

```diff
@@ -6,7 +6,7 @@
 
 ## Common
 ### Trouble on using our toolchain
-When 64 bit machine tries to use 32 bit package like GDB, you can meet below:
```

Comment:
I think it should be GNU tool chain instead of our. We Samsung don't have any propreitory tool chain

Reply:
I missed ```our recommanded toolchain```. But anyway I modified. Thank you for comment.